app/models/user.rb

```diff
@@ -144,9 +144,9 @@ def assignable_roles
 
   def logs_filters(specific_org: false)
     if (support? && !specific_org) || organisation.has_managing_agents?
-      %w[status years user organisation bulk_upload_id]
+      %w[status years assigned_to user organisation bulk_upload_id]
```

Comment:
do we still need user here?

Reply:
Yep - just to distinguish between the "you" selection or the "specific user" selection with the current user selected (both would have the same value for user, but are distinguishable when we also include the assigned_to value